tests/faker/events.py

```diff
@@ -0,0 +1,174 @@
+from faker import Faker
+from faker.providers import BaseProvider
+from tests.faker.events_helper import EventsHelperProvider
+from datetime import datetime
+from tests.test_base import EventType
+import json
+from uuid import uuid4
+
+
+class Event:
+    def __init__(self, EventType, conversation_id: uuid4):
+        self._fake = Faker()
+        self._fake.add_provider(EventsHelperProvider)
+
+        self._conversationId = conversation_id
+        self._eventGeneratedDateTime = datetime.utcnow().isoformat()
+        self._eventType = EventType
+        self._reportingSystemSupplier = self._fake.random_supplier_ods_code()
+        self._reportingPracticeOdsCode = self._fake.random_practice_ods_code()
```

Comment:
_reportingSystemSupplier and _reportingPracticeOdsCode should be set in the specific events class as these fields change depending on which event is being sent / created. i.e. registrations event would have _reportingSystemSupplier = _reqestingSupplierName etc.